applications/GeoMechanicsApplication/tests/Interface/weak_interface_on_beam/MaterialParameters.json

```diff
@@ -32,9 +31,8 @@
               "name"             :  "KratosMultiphysics.StructuralMechanicsApplication.BeamConstitutiveLaw" 
           },
           "Variables": {
-			  "CONSTITUTIVE_LAW_NAME"	 : "KratosMultiphysics.StructuralMechanicsApplication.BeamConstitutiveLaw",
-              "YOUNG_MODULUS"    		 :  1.00E+14,
-              "POISSON_RATIO"    :  0.0,
```

Comment:
What is the reason for these numbers being changed?

Reply:
To have the same Material properties as for the previous test. These data do not affect the solution.

Reply:
It makes sense to have the same numbers in between the tests, but I would expect the results to be different. The test could still pass (since this is the weak one, where only the 1e8 ratio is checked). Did you also check if the actual displacements are also different, or mainly if the test still passes?